### PR TITLE
Remove internal email IDs

### PR DESCRIPTION
**Pull request by abalasa**
#### Problem solved by the commit
Resolve legal scan issues. Removing internal User email ids

### CODE REVIEW

**Review by stsoe — APPROVED** [review]

File: src/CMake/cpackLin.cmake [hunk 1]
```diff
@@ -173,7 +173,6 @@ endif()
 message("-- ${CMAKE_BUILD_TYPE} ${PACKAGE_KIND} package")
 
 SET(CPACK_PACKAGE_VENDOR "Xilinx Inc")
-SET(CPACK_PACKAGE_CONTACT "sonal.santan@xilinx.com")
```
Comment (stsoe) [inline src/CMake/cpackLin.cmake:176]: One question.  Are you sure this is not a required element?

**Comment by keryell** [comment]
What is this?
Do AMD legals are aware of open-source?
Hopefully they do not have discovered the concept of commits and authors... :-(

**Comment by dayeh-xilinx** [comment]
retest this please.

**Comment by dayeh-xilinx** [comment]
retest this please.

**Comment by abalasa** [comment]
Looks like this was already communicated to AMD legal and they are ok with email ids. We can close this PR. Thanks!

**Comment by keryell** [comment]
That said, it might be a good opportunity to update the `@xilinx.com` to `@amd.com` e-mails since at some point they will not work anymore.

**Pull request closed without merging**